templates/layout.jade

```diff
@@ -31,3 +31,15 @@ html(lang='en')
             p &copy; #{ new Date().getFullYear() } #{ locals.owner } &mdash; powered by&nbsp;
               a(href='https://github.com/jnordberg/wintersmith') Wintersmith
               //- please leave the "powered by" if you use the design
+    //- Environment settings
+    - var node_env = process.env.NODE_ENV === 'production' ? 'production' : 'development';
+    script.
+      window.env = {gaId: '#{locals.gaId[node_env]}'};
```

Comment:
It feels weird passing in a camel case string to Jade. Maybe we should make it `locals.ga_id`?